app/controllers/delivery_options_controller.rb

```diff
@@ -0,0 +1,55 @@
+class DeliveryOptionsController < ApplicationController
+  before_action :set_seller
+  before_action :set_seller_delivery_options, only: [:update, :destroy]
+
+  # GET /sellers/:seller_id/delivery_options
+  def index
+    json_response(@seller.delivery_options)
```

Comment:
If you don't expose delivery_type on a different endpoint, you'd have to return it from this endpoint otherwise the frontend won't have any way to get the name and icon_url

Reply:
Would it be better to expose it on a different controller or expose it here? I imagine here? What would that look like? I kind of assumed `@seller.delivery_options` would also expose the delivery type but I guess that's not true?